wordle-web/pages/index.vue

```diff
@@ -1,95 +1,80 @@
 <template>
-  <v-container>
-    <v-progress-linear v-if="game.isBusy" color="primary" indeterminate />
-    <v-card v-else class="text-center">
-      <v-alert
-        v-if="game.gameState != GameState.Playing"
-        :color="game.gameState == GameState.Won ? 'success' : 'error'"
-        class="mb-5"
-        tile
-      >
-        <h3>
-          You've
-          {{ game.gameState == GameState.Won ? "Won! 🥳" : "Lost... 😭" }}
-        </h3>
-        <v-card-text>
-          The word was: <strong>{{ game.secretWord }}</strong>
-        </v-card-text>
-        <v-row v-if="game.stats" class="mb-1" justify="center">
-          <v-col cols="auto">
-            <v-progress-circular
-              size="75"
-              width="10"
-              v-model="game.stats.winPercentage"
-            >
-              {{ game.stats.winPercentage }} %
-            </v-progress-circular>
-            <br />
-            <i class="text-caption">
-              Success Rate
-            </i>
-          </v-col>
-          <v-col cols="auto">
-            <v-progress-circular
-              size="75"
-              width="10"
-              :model-value="game.stats.averageGuessesPercent(game.maxAttempts)"
-            >
-              {{ game.stats.averageGuessesPercent(game.maxAttempts).toFixed(0) }} %
-            </v-progress-circular>
-            <br />
-            <i class="text-caption">
-              Average Guesses
-            </i>
-          </v-col>
-        </v-row>
-        <v-btn variant="outlined" @click="game.startNewGame()">
-          <v-icon size="large" class="mr-2"> mdi-restart </v-icon> Restart Game
-        </v-btn>
-      </v-alert>
-      <v-card-title v-else>Wordle</v-card-title>
-
-      <GameBoardGuess
-        v-for="(guess, i) of game.guesses"
-        :key="i"
-        :guess="guess"
-      />
-
-      <div class="my-10">
-        <Keyboard />
-      </div>
+  <v-container fluid class="image-container">
+      <v-img src="../public/edited-vaporwave-landingPage.jpg">
```

Comment:
I like the image, just gotta put it on the other pages as well

Reply:
Partner took care of it! :)